pkg/dedicatedadmin/dedicatedadmin_test.go

```diff
@@ -0,0 +1,30 @@
+package dedicatedadmin
```

Comment:
Does the test have to go in the package along w/ the source?  My experience is mostly with java (sorry) where tests were off on their own and test resources could be managed independent of the main source.

Reply:
Usually that's the standard I see (even in OS operators), the tests being under the same package as the target code.